package.json

```diff
@@ -52,17 +49,18 @@
     "stylus": "^0.54.8",
     "stylus-loader": "^3.0.2",
     "typescript": "~3.9.3",
-    "vue": "^2.6.11",
+    "vue": "^3.0.0-beta.1",
```

Comment:
Any reason to use this version? I can see there is newers version like `3.0.5`

Reply:
That's the latest version supported by Vue CLI i guess. I just did `vue add vue-next` and got that version.

Reply:
Sound good then